src/microbe_stage/PlacedOrganelle.cs

```diff
@@ -379,6 +388,42 @@ public void ResetGrowth()
         }
     }
 
+    public void ReParentShapes(Microbe to, Vector3 offset)
+    {
+        if (to == currentShapesParent)
+            return;
+
+        var hexes = Definition.GetRotatedHexes(Orientation).ToArray();
+
+        for (int i = 0; i < shapes.Count; i++)
+        {
+            Vector3 shapePosition = Hex.AxialToCartesian(hexes[i]) + Hex.AxialToCartesian(Position);
```

Comment:
This logic and the sizing is duplicated with MakeCollisionShapes, a new method that these both can call should be made instead.